server/command.go

```diff
@@ -600,6 +640,11 @@ func getAutocompleteData() *model.AutocompleteData {
 	webhookAdd.AddTextArgument("[Optional] token: Secret for webhook. Defaults to token used in plugin's settings.", "[token]", "")
 	webhook.AddCommand(webhookAdd)
 
+	webhookBuild := model.NewAutocompleteData(commandBuild, "owner/[repo] reference", "trigger the pipeline")
+	webhookAdd.AddTextArgument("Group or Project path: includes user or group name with optional slash project name", "owner[/repo]", "")
+	webhookAdd.AddTextArgument("reference: reference/branch name for which the pipeline needs to be triggered", "reference", "")
+	webhook.AddCommand(webhookBuild)
```

Comment:
Why is this command a subcommand of `webhook`

Reply:
Oh my bad, seems like a blunder on my side